CFC_DataCollector/recommender/otp.py

```diff
@@ -130,7 +130,7 @@ def turn_into_trip(self, _id, user_id, trip_id, is_fake=None):
 
         mode_list = list(mode_list)
         if is_fake:
-            return Fake_Trip(_id, user_id, trip_id, sections, final_start_time, final_end_time, final_start_loc, final_end_loc, 0, cost, mode_list)
+            return Fake_Trip(_id, user_id, trip_id, sections, start_time, end_time, final_start_loc, final_end_loc)
```

Comment:
why do you want to use start_time and end_time, but final_start_loc and final_end_loc? Please add a comment explaining your decision for future reference.